test/Microsoft.AspNet.Mvc.Core.Test/Filters/ResponseCacheFilterTest.cs

```diff
@@ -9,13 +8,13 @@
 
 namespace Microsoft.AspNet.Mvc
 {
-    public class ResponseCacheAttributeTest
+    public class ResponseCacheFilterTest
     {
         [Fact]
         public void OnActionExecuting_DoesNotThrow_WhenNoStoreIsTrue()
```

Comment:
This does not make sense here any more, it is the attribute that throws not the filter.